### ISSUE TITLE
Feature Request: Matching Cop to `Lint/UnneededDisable`: `Lint/UnneededEnable`

### ISSUE THREAD

**Issue by jfelchner** [bbatsov/rubocop#5356]
The `Lint/UnneededDisable` is very useful, but part of the problem with using it is that, if you wrap code like this:

```ruby
# rubocop:disable Layout/ExtraSpacing
def bar
  puts 'hi'
end
# rubocop:enable Layout/ExtraSpacing
```

Making the `Lint/UnneededDisable` cop happy (either manually or with autocorrection) leaves the `enable` line even though it's no longer needed.

It'd be great to have a cop to handle this use case as well.

**Comment by bbatsov** [comment]
Yeah, that's a good idea. Although it should be named `UndeededCopEnableDirective`. We agreed a while ago that we would rename this `UnneededDisable` to `UnneededCopDisableDirective`, but I guess we forgot to do so.

**Comment by garettarrowood** [comment]
This appears to be a dup of our oldest open issue, #2543 , and #3001 .  It also looks like they got resolved by a new `Lint/MissingCopEnableDirective`. Does this accomplish our goal, or our we not seeing this fire when it should?

**Comment by jfelchner** [comment]
@garettarrowood it's the opposite of `Lint/MissingCopEnableDirective`.  `Lint/MissingCopEnableDirective` is a dupe of #3001.

What I'm asking for is a cop that flags if there's a `rubocop:enable` that is not needed (because the cop(s) were never disabled.

You are correct that this issue is a dupe of #2543.

**Comment by garettarrowood** [comment]
@jfelchner - Thanks for the clarification!

**Comment by garettarrowood** [comment]
Ok. I think I'll tackle this today.
